frontend/packages/employee-frontend/src/components/unit/tab-groups/Groups.tsx

```diff
@@ -159,7 +159,7 @@ export default React.memo(function Groups({
             dataQa="toggle-all-groups-collapsible"
           />
         </TitleContainer>
-        {requireRole(roles, 'ADMIN', 'SERVICE_WORKER', 'UNIT_SUPERVISOR') && (
+        {canManageGroups && (
```

Comment:
Using `canManageGroups` here might not work in the future, but since the endpoint for the data is secured separately, it's not very risky.